src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/ElytraImpactListener.java

```diff
@@ -34,35 +35,31 @@ public ElytraImpactListener(@Nonnull Slimefun plugin) {
 
     @EventHandler
     public void onPlayerCrash(EntityDamageEvent e) {
-        if (!(e.getEntity() instanceof Player)) {
+        if (!(e.getEntity() instanceof Player p)) {
             // We only wanna handle damaged Players
             return;
         }
 
-        if (e.getCause() == DamageCause.FALL || e.getCause() == DamageCause.FLY_INTO_WALL) {
-            Player p = (Player) e.getEntity();
+        if ((e.getCause() == DamageCause.FALL || e.getCause() == DamageCause.FLY_INTO_WALL) && p.isGliding()) {
+            Optional<PlayerProfile> optional = PlayerProfile.find(p);
 
-            if (p.isGliding()) {
-                Optional<PlayerProfile> optional = PlayerProfile.find(p);
-
-                if (!optional.isPresent()) {
-                    PlayerProfile.request(p);
-                    return;
-                }
+            if (optional.isEmpty()) {
```

Comment:
By removing the `isGliding()` check and having `DamageCause#FALL` and `DamageCause#FLY_INTO_WALL` I would imagine this would mean the elytra would now protect you against normal fall damage without activating the elytra. I cant test right now so I will either test tomorrow or if someone else can get here first to test it out, lmk. If it doesn't do this then I am happy with the PR.

Reply:
Derp